lib/stack_master/sparkle_formation/compile_time/empty_validator.rb

```diff
@@ -23,7 +23,7 @@ def has_invalid_values?
         end
 
         def create_error
-          "#{@name} cannot contain empty parameters:#{@parameter.inspect}"
+          "#{@name} is not present inside the 'compile_time_parameters' entry, or has an empty value; found: #{@parameter.inspect}"
```

Comment:
I think we should inform that it's a parameter that's missing. Can sort of infer it from the `compile_time_parameters` name, but better to be explicit.